website/docs/advanced/builder.md

```diff
@@ -0,0 +1,211 @@
+
+:::caution
+
+This feature is currently `not ready` for [Keymanager-API's](../how-prysm-works/keymanager-api.md) use as restarting the validator and beacon node (especially during upgrades) will clear the validator settings for validator registrations. 
+Current persisted values in the db will start but will not be migrated to the cache.
+
+:::
+
+### Parallel Execution
+
+`--build-block-parallel` flag on the beacon node will construct the consensus and execution portions of the beacon block in parallel to improve speed and efficiency. 
+
+### prioritizing local blocks
+
+`--local-block-value-boost` flag is a float64 value that provides an additional percentage to multiply local block value. Use builder block if: `builder_bid_value * 100 > local_block_value * (local-block-value-boost + 100)`. This will encourage your setup to use the local execution if the value earned is not above your threshold, helping to mitigate censorship concerns.
+
+## 3. Is Builder Configured?
+
+When a validator is proposing a block, the following is checked before attempting to use the Builder through the relay.
+- `--http-mev-relay` flag was provided and is pointed to mev-boost or an active relay of the correct network
+- circuit breaker is not triggered 
+- validator is registered (beacon API was successfully called to save to local storage)
+
+If all checks are satisfied then 5. Builder via Relay URL will be used to get the execution payload which contains the transactions and build a blinded block. However, if the checks do not pass then the beacon node will fall back to 4. Local Execution Client.
+
+
+## 4. Local Execution Client: kept in sync and up to date
+
+local execution clients such as `geth` or `nethermind` must continue to run as usual even while using a builder and will be used in case the builder does not pass the `Is Builder Configured?` check. The execution client should be synced and running alongside your beacon node and earnings from the block will be compared to the earnings from the builder's payload. If the local execution payload fails then the entire function will fail.
+
+## 5. Builder: connected via relay URL
+
+The ETHStaker community provides a list of some of the relays that can be used as well as any censorship they may have [here](https://github.com/eth-educators/ethstaker-guides/blob/main/MEV-relay-list.md). You can also run your own locally such as MEV boost but each relay on the list will have their own instructions on how to run. If running your own instead of using a provided url due to latency, you will simply need to update your `--http-mev-relay` flag on your beacon node with the appropriate url for the specific network in use. The relay will connect to a builder which connects to block searchers. 
```

Comment:
```suggestion
The ETHStaker community provides a list of some of the relays that can be used as well as any censorship they may have [here](https://github.com/eth-educators/ethstaker-guides/blob/main/MEV-relay-list.md). You can also run your own locally such as MEV-boost but each relay on the list will have its own instructions on how to run. If running your own instead of using a provided url due to latency, you will simply need to update your `--http-mev-relay` flag on your beacon node with the appropriate url for the specific network in use. The relay will connect to a builder which connects to block searchers. 
```